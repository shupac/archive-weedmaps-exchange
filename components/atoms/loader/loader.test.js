import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Loader from './index';

describe('The Loader', () => {
  it('will render without a provided title ', () => {
    const tree = toJson(shallow(<Loader />));
    expect(tree).toMatchSnapshot();
  });
  it('will render with a provided title ', () => {
    const tree = toJson(shallow(<Loader loaderText="Loader text" />));
    expect(tree).toMatchSnapshot();
  });
});
