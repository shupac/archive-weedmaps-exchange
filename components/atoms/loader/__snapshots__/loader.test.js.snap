// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loader will render with a provided title  1`] = `
<loader__LoaderWrap>
  <Spinner
    fill="#727682"
    size="70px"
  />
  <p>
    Loader text
  </p>
</loader__LoaderWrap>
`;

exports[`The Loader will render without a provided title  1`] = `
<loader__LoaderWrap>
  <Spinner
    fill="#727682"
    size="70px"
  />
</loader__LoaderWrap>
`;
